### PR TITLE
Delete send confirmation

### PR DESCRIPTION
**Pull request by dbo1dyrev**
Here I delete old SendConfirmation screen and all depended components and actions.

### CODE REVIEW

File: src/actions/ScanActions.js [hunk 1]
```diff
@@ -149,7 +148,7 @@ export const parseScannedUri = (data: string, customErrorTitle?: string, customE
 
       if (isLegacyAddressUri(parsedUri)) {
         // LEGACY ADDRESS URI
-        return dispatch(showLegacyAddressModal())
+        // BUSTED!
```
Comment (swansontec) [inline src/actions/ScanActions.js:151]: We cannot leave these two in a busted state.

File: src/actions/ScanActions.js [hunk 2]
```diff
@@ -159,8 +158,7 @@ export const parseScannedUri = (data: string, customErrorTitle?: string, customE
 
       if (isPaymentProtocolUri(parsedUri)) {
         // BIP70 URI
-        // $FlowFixMe
-        return dispatch(paymentProtocolUriReceived(parsedUri))
+        // BUSTED!
```
Comment (swansontec) [inline src/actions/ScanActions.js:161]: We cannot leave these two in a busted state. We need to somehow route these two URI's to the new Send scene. This is the reason we haven't already removed the SendConfirmation scene - doing so would break deep linking.

**Review by swansontec — CHANGES_REQUESTED** [review]

File: src/actions/ScanActions.js [hunk 3]
```diff
@@ -120,90 +123,105 @@ export const parseScannedUri = (data: string, customErrorTitle?: string, customE
...
-          walletId: selectedWalletId,
-          wallet: guiWallet,
-          onAddToken: noOp
-        }
-        return Actions[ADD_TOKEN](parameters)
+  try {
+    const parsedUri: EdgeParsedUri = await edgeWallet.parseUri(data, currencyCode)
+    dispatch({ type: 'PARSE_URI_SUCCEEDED', data: { parsedUri } })
+
+    console.log('parsedUri parsedUri parsedUri parsedUri parsedUri', parsedUri)
```
Comment (swansontec) [inline src/actions/ScanActions.js:130]: Delete console.log

File: src/components/themed/Fade.js [hunk 4]
```diff
@@ -27,6 +27,7 @@ const FadeComponent = ({ visible: propsVisible, hidden, children }: Props) => {
     })
   }
 
+  // eslint-disable-next-line
```
Comment (swansontec) [inline src/components/themed/Fade.js:30]: These should be `eslint-disable-next-line rule-name`, so we know what rule is being switched off. Also, these cleanups should be in a separate commit.

**Review by swansontec — CHANGES_REQUESTED** [review]
I think this approach works well, but I don't like the way the commits are organized. I would like to see 3 commits with these names:

- Clean up eslint warnings
- Use the new send screen for deep links
- Delete the send confirmation screen

So, this will take a bit of rebasing to get right.

**Comment by thehobbit85** [comment]
Also, resolve the conflicts

**Review by swansontec — APPROVED** [review]
The code looks OK, but the commits are disorganized. It would be preferable to squash these, at this point, rather than merge them as-is.

**Review by paullinator — CHANGES_REQUESTED** [review]
Agree with @swansontec on this one. Please clean up the commits before this is merged.

**Review by swansontec — CHANGES_REQUESTED** [review]
This is still totally wrong, so I'll do the rebase myself, so we are perfectly clear about *what* a clean patch series looks like, then submit a new PR. Dennis can then review my PR, to get an idea for what we are looking for here.

**Comment by swansontec** [comment]
Replaced by #2626, which has the same changes rebased into a series of standalone commits, instead of jumbled together like in this branch.
